flake.nix

```diff
@@ -33,6 +33,7 @@
               yq-go
               kubernetes-helm
               opentofu
+              k9s
```

Comment:
same here

Reply:
This is a tiny improvement doing as a sidetask. I guess i should have mentioned it in the description?